packages/wallets/jest.config.js

```diff
@@ -10,4 +10,5 @@ module.exports = {
         '\\.(ts|tsx)?$': '<rootDir>/jestTransformer.js',
         '^.+\\.jsx?$': 'babel-jest',
     },
+    transformIgnorePatterns: ['/node_modules/(?!(@deriv-com/ui)).+\\.js$'],
```

Comment:
@wojciech-deriv  This is needed to ignore the node modules coming from the deriv-com/ui package and allowing the tests to run. Removing this would cause this error when running the tests:
<img width="954" alt="image" src="https://github.com/binary-com/deriv-app/assets/142860499/aa67c15b-679f-4343-8454-b5434575ee17">